src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -91,8 +92,6 @@ protected virtual Task Stop()
 
     async Task CreateSeedData(ICreateSeedData instance)
     {
-        Log.Info("Creating or purging queues...");
-        await CreateOrPurgeAndDrainQueues().ConfigureAwait(false);
         Log.Info("Creating send only endpoint...");
```

Comment:
Ah, now I get it... Thanks for clarifying @ramonsmits. I think three dots will do just fine 😸